cmd/exec/operations.go

```diff
@@ -497,22 +516,26 @@ func executeEvent(url string) (string, error) {
 		)
 
 		if err != nil {
-			return event["id"].(string), errors.New("filtetype not json")
+			return "", errors.New("direktiv server not reachable")
```

Comment:
This error message is not always accurate. Actually, it may always be incorrect. Line 524 is where the request is actually attempted. It also half-suppresses the error -- it doesn't let the program fail quietly, but it does hide whatever the real error was.